src/api/post.js

```diff
@@ -0,0 +1,21 @@
+import { client } from "./httpClient";
+
+export const getUserPosts = (userId) => {
+  return client.get(`/posts?userId=${userId}`);
+};
+
+export const getPosts = () => {
+  return client.get("/posts");
+};
+
+export const createPosts = ({ body, title, userId }) => {
```

Comment:
It seems there is a minor naming issue here. Your function is named `createPosts`, but it appears to create only a single post. Consider renaming this function to `createPost` for clarity.